hw09-orm/src/test/java/ru/otus/homework/JdbcTemplateTest.java

```diff
@@ -0,0 +1,136 @@
+package ru.otus.homework;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ru.otus.homework.api.dao.ObjectDao;
+import ru.otus.homework.api.model.Account;
+import ru.otus.homework.api.model.User;
+import ru.otus.homework.api.service.DbService;
+import ru.otus.homework.api.service.DbServiceImpl;
+import ru.otus.homework.jdbc.dao.ObjectDaoJdbc;
+import ru.otus.homework.jdbc.sessionmanager.SessionManagerJdbc;
+
+import javax.sql.DataSource;
+import java.math.BigDecimal;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JdbcTemplateTest {
+    private DbService dbService;
+    private DataSource dataSource;
+    private User user1 = new User(1, "name1", 20);
```

Comment:
```suggestion
    private User user1 = new User("name1", 20);
```

Id не нужно присваивать объекту при его создании.
Id у вас автоматически генерируется в БД (auto_increment).
```sql
create table User(id bigint(20) NOT NULL auto_increment, name varchar(255), age int(3))
```

Надо после сохранения объекта в БД получить сгенерированный id и присвоить это значение в поле id объекта.

Reply:
Поменяла возвращаемое значение для метода сохранения объекта, чтобы можно было по возвращаемому id искать объект. В задании было create(T objectData)